src/sentry/static/sentry/app/components/activity/feed.jsx

```diff
@@ -47,7 +47,13 @@ class ActivityFeed extends React.Component {
       location.pathname != nextLocation.pathname ||
       location.search != nextLocation.search
     ) {
-      this.remountComponent();
+      this.setState({
+        itemList: [],
+        loading: true,
+        error: false,
+        pageLinks: null,
+      });
+      this.fetchData();
```

Comment:
Should this hook be replaced with a `componentDidUpdate`?

Reply:
They probably all should in the long term since `componentWillReceiveProps` is getting deprecated, but I don't want to tackle that here

Reply:
Or did you mean something else?